pkg/generators/models/testdata/cases/required_properties/expected/model_foo.go

```diff
@@ -5,12 +5,21 @@
 //     Version: 0.1.0
 package generatortest
 
+import (
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+)
+
 // Foo is an object.
 type Foo struct {
 	// Bar:
 	Bar string `json:"bar"`
 }
 
+// Validate implements basic validation for this model
+func (m Foo) Validate() error {
+	return validation.Errors{}.Filter()
```

Comment:
wouldn't it be cleaner to just return `nil`?

Reply:
That would require even more logic ( I would have to decide for the whole struct and not only for the props if validation is required. Yes, it would be cleaner, but the rise in complexity on the other hand is really avoidable by living with that here